lib/memory/memory.dart

```diff
@@ -25,7 +26,10 @@ class MemoryScreen extends Screen {
   StatusItem objectCountStatus;
 
   PButton loadSnapshotButton;
-  Table<ClassHeapStats> memoryTable;
+
+  // TODO(dantup): Is it reasonable to put dynamic here?
```

Comment:
`Object` may be more correct here - the common super class of all the things we could display.